src/Microsoft.Health.Dicom.Api/Registration/DicomServerServiceCollectionExtensions.cs

```diff
@@ -103,6 +106,16 @@ public static IDicomServerBuilder AddBackgroundWorkers(this IDicomServerBuilder
                 c.UseApiBehavior = false;
             });
 
+            services.AddApiVersioning();
```

Comment:
`AddApiVersioning()` has already been added above - do we need it again?

Reply:
removed, thanks